commit 5837efa6d43ee235ab254efc5fe383b0837affae
Author: NexAdn <nex@nexadn.de>
Date:   Mon Sep 18 09:57:42 2023 +0200

    Makefile: add EXTRA_LFLAGS to linker command line
    
    This allows custom linker flags to be set by users/packagers (e.g. to
    switch the linker used for linking the software).
    
    See-Also: https://gitlab.fem-net.de/gentoo/fem-overlay/-/issues/66

--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 DEBUGopts = -g -O0 -fno-inline-functions -DDEBUG
 NDEBUGopts = $(EXTRA_CFLAGS) -O2 -DNDEBUG
 CFLAGS = -Wall -c $(DEBUG) -D_FILE_OFFSET_BITS=64 -DFUSE_USE_VERSION=25
-LFLAGS = -Wall -lmxml -lfuse $(DEBUG)
+LFLAGS = -Wall -lmxml -lfuse $(DEBUG) $(EXTRA_LFLAGS)
 CC = gcc
 DEBUG=$(NDEBUGopts)
 
